libmscore/barline.cpp

```diff
@@ -1272,10 +1272,11 @@ QString BarLine::accessibleExtraInfo() const
                   for (const Element* e : nextM->el()) {
                         if (!score()->selectionFilter().canSelect(e))
                               continue;
-                        if (e->isMarker())
+                        if (e->isMarker()) {
                               if (toMarker(e)->markerType() == Marker::Type::FINE)
                                     continue; //added above^
                               rez = QString("%1 %2").arg(rez).arg(e->screenReaderInfo());
+                              }
```

Comment:
Looks like either way works, just differently.  The way it actually was, the screenreader would read *all* elements attached to the next measure, except for Markers of type Fine.  The way it *looks* like it was, it would only read *markers*, again except for ones of type Fine.

Meaning, with a barline selected, if something was attached to the next measure that was *not* a marker, it would be read, and the proposed changed would stop that.  Without knowing the full list of things that could conceivably be attached to measures, it's hard to say whether you'd *want* other things read or not.  Since we don't read anything else attached to *this* measure except jumps and Fine markers, I am inclined to think we don't want things attached to the *next* measure read either.

Bigger picture: I guess the point here is to read "coda" when selecting the barline before the coda.  This really only makes sense if the coda measure directly follows the previus measure, no intervening line break or horizontal frame (because otherwise the barline at the end of the measure before the coda would not be physically attached to the coda).  As it is, it looks like we'll actually read "coda" when selecitng the barline at the end of a line if the next system starts with a coda.  I'm not sure that makes sense.  So we could also consider removing this whole loop.

Bottom line: it's going to work one way or another, so it doesn't matter much.  @andreituicu feel free to weigh in if you can remember your reasoning.

Right now, my inclination is to remove the whole loop.  Maybe replace this section of code with something that really figures out if this barline is at the start or end of the measure and reads accordingly.  But I also want to implement a command to navigate all elements directly, meaning we eventually won't want to read anything else attached to the measure just because a barline is selected.